mozilla-release/mobile/android/base/java/org/mozilla/gecko/home/BookmarkFolderView.java

```diff
@@ -72,6 +72,10 @@ public BookmarkFolderView(Context context) {
     public BookmarkFolderView(Context context, AttributeSet attrs) {
         super(context, attrs);
 
+        /* Cliqz Start */
+        setOrientation(LinearLayout.VERTICAL);
```

Comment:
please take a look to my comment in merge block, and handle the vertical orientation in xml.

Reply:
Did that. Thanks!